app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminDisConfNewsletterEnableRegConfirmActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminDisConfNewsletterEnableRegConfirmActionGroup">
```

Comment:
AG name should be descriptive and meaningful without any contracted words
I really don't understand what this AG actually do if see only name